v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/schema/Schema.java

```diff
@@ -30,13 +30,13 @@
  */
 public class Schema implements Serializable {
   /** Maps the HarbourBridge table ID to the Spanner table details. */
-  private final Map<String, SpannerTable> spSchema;
+  private Map<String, SpannerTable> spSchema;
```

Comment:
not sure if this change is required. It should be possible to generate the srcSchema up front and set it along with the constructu.

Reply:
@bharadwaj-aditya we are generating in two different place so it will be become hard to accomodate in on place